### PR TITLE
update istio api

### PR DESCRIPTION
**Pull request by yangminzhu**

### CODE REVIEW

File: pilot/pkg/model/jwks_resolver.go [hunk 1]
```diff
@@ -197,6 +197,7 @@ func (r *JwksResolver) SetAuthenticationPolicyJwksURIs(policy *authn.Policy) err
 	for _, method := range policy.Peers {
 		switch method.GetParams().(type) {
 		case *authn.PeerAuthenticationMethod_Jwt:
+			// nolint: staticcheck
```
Comment (incfly) [inline pilot/pkg/model/jwks_resolver.go]: why we need these changes? are they auto generated as well?

Comment (yangminzhu) [reply pilot/pkg/model/jwks_resolver.go]: No, it's not auto generated, the lint will fail without this.

Comment (howardjohn) [reply pilot/pkg/model/jwks_resolver.go]: its because we deprecated these fields

**Review by howardjohn — APPROVED** [review]

**Review by diemtvu — APPROVED** [review]

**Pull request merged**